typings/index.d.ts

```diff
@@ -246,25 +246,22 @@ declare class Command extends NodeJS.EventEmitter {
     usage(): string;
 
     /**
-     * Get the name of the command.
+     * Get or set the name of the command.
      *
-     * @return {string}
+     * @param {string} str
+     * @return {string|Command}
      */
-    name(): string;
+    name(str?: string): string | Command;
 
     /**
      * Output help information for this command.
      *
-     * @param {(str: string) => string} [cb] Callback method
+     * @param {() => void} [cb] Callback method
      */
-    outputHelp(cb?: (str: string) => string): void;
+    outputHelp(cb?: () => void): void;
```

Comment:
Callback takes a string as the first parameter.

Reply:
fixed